LoopKitUI/ViewModels/TherapySettingsViewModel.swift

```diff
@@ -11,37 +11,37 @@ import LoopKit
 import HealthKit
 import SwiftUI
 
-public typealias SyncSchedule = (_ items: [RepeatingScheduleValue<Double>], _ completion: @escaping (Swift.Result<BasalRateSchedule, Error>) -> Void) -> Void
```

Comment:
Thanks for moving these to an even more appropriate spot, now that the delegate is defined.